src/vscode-dts/vscode.proposed.interactive.d.ts

```diff
@@ -115,6 +115,10 @@ declare module 'vscode' {
 		errorDetails?: InteractiveResponseErrorDetails;
 	}
 
+	export interface InteractiveContentReference {
+		reference: Uri | Location;
```

Comment:
Is a `Location` enough for @mattbierner's case of pointing at a symbol? Would you expect that range to be updated as the file is edited, in which case it would have to be computed by the client on demand?

Reply:
`Location` should be enough, at least that's what other language features like go to def and find refs use. 

> Would you expect that range to be updated as the file is edited, in which case it would have to be computed by the client on demand?

Yeah, likely expected but IMO the client should do that. It could register an editor/model decoration which gets tracked automatically. Extensions won't be able to do it with the same fidelity

Reply:
I wonder if showing references would benefit from indicating document versions as well